LoggerConfig/SerilogConfiguration.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuickEdit.LoggerConfig;
+
+public class SerilogConfiguration
+{
+    public static void ConfigureLogger()
+    {
+        var intermediateOutputPath =
+            Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..", @"obj\Debug\net8.0\Logs"));
+
+        Directory.CreateDirectory(intermediateOutputPath);
+
+        var logPath = Path.Combine(intermediateOutputPath, $"consoleapplog-{DateTime.UtcNow:yyyyMMddHHmmss}.txt");
+
+        Log.Logger = new LoggerConfiguration()
+            .MinimumLevel.Information()
+            .Enrich.WithMachineName()
+            .WriteTo.Console()
+            .WriteTo.File(logPath, rollingInterval: RollingInterval.Day)
+            .CreateLogger();
```

Comment:
The logger should probably log debug messages when debug is on in the configuration

I'm not very familiar with Serilog just yet, so my implementation of this might not be very good, but this is what I came up with:
```suggestion
	var loggerConfig = new LoggerConfiguration()
		.Enrich.WithMachineName()
		.WriteTo.Console()
		.WriteTo.File(logPath, rollingInterval: RollingInterval.Day);

	if (Program.config != null && Program.config.debug)
		loggerConfig.MinimumLevel.Debug();
	else
		loggerConfig.MinimumLevel.Information();
		
	Log.Logger = loggerConfig.CreateLogger();
```